years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if (grep -r "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . | grep -v 2009-`date +%Y`) then
```

Comment:
@vkuchyn sorry if I was unclear here, @mkordas is right :), all I was trying to get across was to grep files in a way that is robust against the number of whitespaces between the words ( say between "Copyright" and "2009" ) when constraining the result set :) Simply because we have copyright information in source files in multiple languages => it's not impossible to have some issues with that :)